src/views/Registration/RegistrationDefineBusiness.vue

```diff
@@ -9,7 +9,19 @@
       <v-card flat class="mt-5">
         <NameRequestInfo />
 
-        <!-- FUTURE: Business Type goes here -->
+        <template v-if="isTypePartnership">
+          <v-divider class="mx-6" />
+
+          <!-- Business Type -->
+          <header id="business-type-header" />
+          <BusinessTypeGP
+            class="py-8 px-6"
+            :class="{ 'invalid-section': getShowErrors && !businessTypeGPValid }"
+            :showErrors="getShowErrors"
+            @update:businessTypeGP="setRegistrationBusinessTypeGP($event)"
```

Comment:
Do you need to restore the value from a draft filing (and thus pass it in to this component)?

Reply:
I need to restore the value from the draft filing, and I've updated the code. Please review it.

Reply:
Did you know that you could make your component support a `v-model` prop?

Ref: https://v2.vuejs.org/v2/guide/components.html?redirect=true#Using-v-model-on-Components

TLDR: if you component accepts a `value` prop and emits an `input` event, then the parent (THIS page) can simply use `v-model="yourLocalVariableName"`.

The reason we generally don't do this is that we need a getter/action to read/write from/to the store. So then we would need to separate the v-model's input and output behaviour, maybe like this:
```
get yourLocalVariableName (): boolean {
  // put store getter call here
}
set yourLocalVariableName (val: boolean): void {
  // put store action call here
}
```
As you can see, this doesn't save any code at all. In fact I think it's more complicated to understand, and I'm all for simple-to-understand code :)

Reply:
Closing this conversation because some of the code has been updated.